### DISCUSSION TITLE
How to survive a system reboot?

### DISCUSSION THREAD

**Discussion by HubKing** [sezanzeb/input-remapper#867] [Q&A] [answered]
It works when I click Apply in the GUI, but after a reboot, it does not work. I have to click Apply again. How can I make the applied preset work even after a system reboot?

I saw https://github.com/sezanzeb/input-remapper/blob/HEAD/readme/usage.md but I could not find the information.

**Comment by lucentwolf** [comment, marked as answer]
HubKing, did you activate the Autoload toggle?

![image](https://github.com/sezanzeb/input-remapper/assets/60614876/58daf0e2-0562-4f44-a85a-4054130d6506)

**Reply by HubKing** [reply]
Checking it did it, but why are those so dim? I had not noticed the toggle.

![image](https://github.com/sezanzeb/input-remapper/assets/10900489/efe1088d-c620-44d1-b251-b9791c64fc3d)

**Reply by TheOverpassArsonist** [reply]
For what it's worth I quietly second the point about it being too dim. I'm not sure if the application even has control over it or if it's a constant within GTK that just looks worse due to the colors chosen for the application but to be honest when I first saw it I thought it was grey-ed out and I silently erased it mentally. 

I am by no means a UI designer but I would wager that the reason it feels greyed out is that the "Rename" and "Autoload" text is greyed as well, as opposed to the white text elsewhere in the application.
